pypeclub/OpenPype
# Traypublisher: simple editorial publishing

## Brief description
Adding editorial publishing to traypublisher.

## Description
Converting standalone editorial family publishing (simple workflow) to traypublisher with additional features for individual or grouped instances attributes configuration.

## Additional info
- creator with attributes for precise navigation
- clip instances supporting multiple selection and editing
- avoid settings inputs as much as possible

## Documentation (add _"type: documentation"_ label)
WIP [feature_documentation](future_url_after_it_will_be_merged)

## Testing Todo:
1. use data for testing from here https://drive.google.com/drive/folders/1iMAcgRvt6Px-LCwJmwhJDi0qBSaPf7Cn
2. Open Pyblisher (new) from tray and set project. And select `Editorial Simple` and folder in project tree.
![image](https://user-images.githubusercontent.com/40640033/180226859-bfef5b34-84f2-446f-b22a-386d787440fb.png)
3. just in case your project is not 25fps set it in inputs, also toggle on all subsets
![image](https://user-images.githubusercontent.com/40640033/180227207-bfd90520-a3b3-4051-ab60-4e7e43730559.png)
4. you can also set different `workfile start frame` or handles > those can be changed later in instance attributes on shot instances. 
5. Hit publish and see that all are created in ftrack and your DB
